src/ontology/reports/icd10who.synonyms.updated.robot.tsv

```diff
@@ -1,3 +1,3 @@
-mondo_id	mondo_label	synonym_scope	synonym_scope_mondo	synonym	synonym_exact	synonym_broad	synonym_narrow	synonym_related	source_id	source_label	synonym_type	mondo_evidence	case
-ID					A oboInOwl:hasExactSynonym	A oboInOwl:hasBroadSynonym	A oboInOwl:hasNarrowSynonym	A oboInOwl:hasRelatedSynonym	>A oboInOwl:hasDbXref		>AI oboInOwl:hasSynonymType		
-MONDO:0004985	bipolar disorder	oio:hasExactSynonym	oio:hasRelatedSynonym	bipolar affective disorder	bipolar affective disorder				ICD10WHO:F31	Bipolar affective disorder			
+mondo_id	mondo_label	synonym_scope	synonym_scope_mondo	synonym	synonym_exact	synonym_broad	synonym_narrow	synonym_related	source_id	source_label	synonym_type	synonym_type_mondo	mondo_evidence	case
```

Comment:
- `synomym` column is a bit redundant (if you wanted to make space), no biggie if not
- Did you make sure that robot correctly adds the evidence even if the synonym column is not directly preceding the evidence column?

Reply:
I didn't get to check that yet.

As discussed, the synonym and synonym_scope columns are here to make curation easier but if you think that is not the case I can remove.